dummyintent/src/main/java/org/openbase/bco/psc/dummyintent/DummyIntentController.java

```diff
@@ -0,0 +1,156 @@
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import org.openbase.bco.psc.dummyintent.rsb.RSBConnection;
+import org.openbase.bco.registry.remote.Registries;
+import org.openbase.jul.exception.CouldNotPerformException;
+import org.openbase.jul.exception.InitializationException;
+import org.openbase.jul.exception.NotAvailableException;
+import org.openbase.jul.exception.printer.ExceptionPrinter;
+import org.openbase.jul.exception.printer.LogLevel;
+import org.openbase.jul.extension.rsb.com.RSBFactoryImpl;
+import org.openbase.jul.extension.rsb.iface.RSBInformer;
+import org.openbase.jul.iface.Launchable;
+import org.openbase.jul.iface.VoidInitializable;
+import org.openbase.type.domotic.unit.UnitProbabilityCollectionType;
+import org.slf4j.LoggerFactory;
+import rsb.AbstractEventHandler;
+import rsb.Event;
+import rst.dialog.SpeechHypothesisType.SpeechHypothesis;
+
+/**
+ * This class controls the dummy intent
+ *
+ * @author <a href="mailto:dreinsch@techfak.uni-bielefeld.de">Dennis Reinsch</a>
+ * @author <a href="mailto:jbitschene@techfak.uni-bielefeld.de">Jennifer Bitschene</a>
+ * @author <a href="mailto:jniermann@techfak.uni-bielefeld.de">Julia Niermann</a>
+ *
+ */
+public class DummyIntentController extends AbstractEventHandler implements DummyIntent, Launchable<Void>, VoidInitializable {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(DummyIntentController.class);
+    private RSBConnection rsbConnection;
+    private boolean initialized = false;
+    private boolean active = false;
+    private RSBInformer<UnitProbabilityCollectionType.UnitProbabilityCollection> unitInformer;
+
+    @Override
+    public void handleEvent(final Event event) {
+        LOGGER.trace(event.toString());
+
+        if (event.getData() instanceof String) {
+
+            String intent = (String) event.getData(); // should be something like colorstate[color:blau]
+
+            try {
+
+                if (intent.contains("unit")) {
+                    intent = intent.replaceFirst("unit", "");
+
+                    // publish unit for testing
+                    UnitProbabilityCollectionType.UnitProbabilityCollection.Builder collectionBuilder = UnitProbabilityCollectionType.UnitProbabilityCollection.newBuilder();
+                    collectionBuilder.addElementBuilder().setId("47e63f5a-ff30-4b0d-905a-815f94aa8b50").setProbability(1.0f);
```

Comment:
Please avoid hardcoding any ids. Otherwise this only works within the apartment. A more flexible solution would be to resolve the unit id via its type through the registry e.g. ``Registries.getUnitRegistry().getUnitConfigsByUnitType(UnitTemplateType.UnitTemplate.UnitType.COLORABLE_LIGHT).get(0).getId()``